src/main/java/minegame159/meteorclient/modules/misc/Baconator.java

```diff
@@ -0,0 +1,130 @@
+package minegame159.meteorclient.modules.misc;
+
+import meteordevelopment.orbit.EventHandler;
+import minegame159.meteorclient.events.world.TickEvent;
+import minegame159.meteorclient.modules.Categories;
+import minegame159.meteorclient.modules.Module;
+import minegame159.meteorclient.settings.BoolSetting;
+import minegame159.meteorclient.settings.DoubleSetting;
+import minegame159.meteorclient.settings.EntityTypeListSetting;
+import minegame159.meteorclient.settings.IntSetting;
+import minegame159.meteorclient.settings.Setting;
+import minegame159.meteorclient.settings.SettingGroup;
+import minegame159.meteorclient.utils.Utils;
+import minegame159.meteorclient.utils.player.InvUtils;
+
+import net.minecraft.block.Block;
+import net.minecraft.block.Blocks;
+import net.minecraft.entity.Entity;
+import net.minecraft.entity.EntityType;
+import net.minecraft.entity.LivingEntity;
+import net.minecraft.item.FlintAndSteelItem;
+import net.minecraft.item.Items;
+import net.minecraft.util.Hand;
+import net.minecraft.util.hit.BlockHitResult;
+import net.minecraft.util.math.Direction;
+import net.minecraft.util.math.Vec3d;
+
+import it.unimi.dsi.fastutil.objects.Object2BooleanMap;
+
+public class Baconator extends Module {
+    private final SettingGroup sgGeneral = settings.getDefaultGroup();
+
+    private final Setting<Double> distance = sgGeneral.add(new DoubleSetting.Builder().name("distance")
+            .description("The maximum distance the aniaml has to be to be roasted.").min(0.0).defaultValue(5.0)
```

Comment:
english
```suggestion
            .description("The maximum distance the animal has to be to be roasted.").min(0.0).defaultValue(5.0)
```

Reply:
English indeed, but i don't think they will merge it anyway